README.md

```diff
@@ -291,6 +284,71 @@ One other tip... This totally works:
 </glamorous.Div>
 ```
 
+#### Overriding component styles
```

Comment:
Above it's mentioned that the three ways props are `css`, `className`, and `theme`. Despite the fact that `theme` is documented elsewhere, I think it would be valuable to add a little documentation here as well (even if it's just a mention).